app-ios/tutanota/Sources/Alarms/RepeatRule.swift

```diff
@@ -56,5 +82,10 @@ extension RepeatRule {
       return try decrypt(base64: endValue, key: sessionKey)
     }
     self.endCondition = RepeatEndCondition(endType: endType, endValue: endValue ?? 0)
+    let decryptedExclusions: [Date] = try encrypted.excludedDates.map { encWrapper in
+      let timestamp: Int64 = try decrypt(base64: encWrapper.date, key: sessionKey)
```

Comment:
don't we have a function to decrypt a date?

Reply:
yes